### PR TITLE
Pin multidict dependency for aiobotocore02 tests

### PR DESCRIPTION
**Pull request by codeboten**
The new version of multidict released yesterday (4.6.0) causes failures in aiobotocore2 tests. Pinning version as a workaround. Note: I ran into this problem only once the build cache had been invalidated.

Signed-off-by: Alex Boten <aboten@lightstep.com>

### CODE REVIEW

**Comment by jd** [comment]
Can you maybe rather use `!=4.6.0`?

File: tox.ini [hunk 1]
```diff
@@ -157,6 +157,7 @@ deps =
     aiobotocore04: aiobotocore>=0.4,<0.5
     aiobotocore03: aiobotocore>=0.3,<0.4
     aiobotocore02: aiobotocore>=0.2,<0.3
+    aiobotocore02: multidict==4.5.2
```
Comment (jd) [inline tox.ini]:
```suggestion
    aiobotocore02: multidict!=4.6.0
```

Comment (codeboten) [reply tox.ini]: Just tested it with the release of 4.6.1 from hours ago and it's still failing. If time permits, I'll try to look into the changes https://github.com/aio-libs/multidict/compare/v4.5.2...v4.6.0

Comment (brettlangdon) [reply tox.ini]: Do you have what the failure is? Probably something we should make sure we support.

Comment (codeboten) [reply tox.ini]: I put the log in a gist https://gist.github.com/codeboten/6aeb044cd1578790bfbc128378eb2d79

Comment (jd) [reply tox.ini]: Yeah that looks like a bug in our (test) code. We need to fix it.

Comment (jd) [reply tox.ini]: I actually misread the data. This is triggered by a test but it's not our code, it's just that this old version of aiobotocore cannot work with the new multidict release.
It means that nobody can install this version of aiobotocore without some requirements hack.

Comment (majorgreys) [reply tox.ini]: Why not go with `multidict<=4.6.0` in this case?

**Pull request merged**